pyalgotrading/algobulls/connection.py

```diff
@@ -501,9 +506,13 @@ def start_job(self, strategy=None, start=None, end=None, instruments=None, lots=
                     instrument_list.append({'id': _["id"]})
                     break
 
-        # delete previous trades
-        if delete_previous_trades and trading_type in [TradingType.BACKTESTING, TradingType.PAPERTRADING]:
-            self.delete_previous_trades(strategy)
+        # save BT/PT parameters
+        if trading_type in [TradingType.BACKTESTING, TradingType.PAPERTRADING]:
+            self.saved_params[strategy] = {'parameters': parameters, 'candle': candle, 'instruments': instruments, 'mode': mode, 'lots': lots, 'initial_funds_virtual': initial_funds_virtual, 'vendor_details': broking_details}
+
+            # delete previous trades
+            if delete_previous_trades:
+                self.delete_previous_trades(strategy)
 
         # Setup config for starting the job
         strategy_config = {
```

Comment:
Let's print the proper config before starting a job, so a user understands what parameters is the job running for. Find appropriate place for the same, this is a dummy location,

Reply:
done